src/targets/services/cleanOrphanAccounts.js

```diff
@@ -0,0 +1,81 @@
+const cozyFetch = require('../../lib/services/cozyFetch')
+
+const logMessagePrefix = level =>
+  `[cozy-collect][cleanOrphanAccounts][${level}]:`
+
+const output = level =>
+  function() {
+    console.log.apply(
+      console,
+      [logMessagePrefix(level)].concat(
+        // Node < 7 does not know about Object.values
+        Object.keys(arguments).map(key => arguments[key])
+      )
+    )
+  }
+
+const log = {
+  error: output('error'),
+  info: output('info')
+}
+
+const handleFetchError = error => log.error(error.message)
+
+log.info('Fetching accounts...')
+
+cozyFetch(
+  'GET',
+  '/data/io.cozy.accounts/_all_docs?include_docs=true',
```

Comment:
Hum, be careful here. This route will also return you indexes created for this doctype (couchDB behaviour). So, since they don't match any trigger, I think you will remove indexes with this script.
Here is how `cozy-client-js` [filter indexes](https://github.com/cozy/cozy-client-js/blob/master/src/data.js#L105), maybe you can use the same RegExp for this script also.

Reply:
Well spotted, thanks !